README.md

```diff
@@ -0,0 +1,59 @@
+# My Public IP
+
+## About The Project
+This program will give you, your **Public IP**
+
+### Build With
+* Python
+
+## Getting Started
+Go to the Project Directory
+
+### Prerequisites
+For installing setup.py you need ```setuptools``` python library
+If you don't have this library on your system, try the command below
+
+```pip install setuptools```
+
+### Installation
+1. While you're in Project Directory, write the message below in the terminal
+
+    ```make install```
+
+   *Note* : You can also use ```sudo make install```
+2. Whenever you need this project in your program, just import it 
+
+    ```import my_ip_public```
+3. You can call these three functions to get your public IP from 3 diffrent providers
+
+* [```my_ip_public.ipify_org()```](https://ipify.org)
+* [```my_ip_public.my_ip_com()```](https://myip.com)
+* [```my_ip_public.my_ip_io()```](https://my-ip.io)
+
+4. You can always clean this, by ```make clean``` command
+
+### Usage
+This is the example of how to use them
+```
+muhammadmahdi:~/MyFiles/git/13/my_ip_public$ python3
```

Comment:
please remove your name and set a generic path name

Reply:
Updated